root_s3_bucket_access.tf

```diff
@@ -1,8 +1,9 @@
 # AWS SSO groups that require access to encrypted s3 buckets need updating with relevant decrypt permissions for KMS s3 Key
 
 locals {
-  aws_sso_export_bucket_access_roles   = [data.aws_ssm_parameter.aws_sso_admin_role.value, data.aws_ssm_parameter.aws_sso_export_role.value]
-  aws_sso_internal_bucket_access_roles = [data.aws_ssm_parameter.aws_sso_admin_role.value]
+  aws_sso_export_bucket_access_roles_non_prod   = local.environment != "prod" ? [data.aws_ssm_parameter.aws_sso_admin_role.value, data.aws_ssm_parameter.aws_sso_export_role.value] : []
```

Comment:
This change can be simplified to: 
```
aws_sso_export_bucket_access_roles   = local.environment == "prod" ? [] : [data.aws_ssm_parameter.aws_sso_admin_role.value, data.aws_ssm_parameter.aws_sso_export_role.value]
  aws_sso_internal_bucket_access_roles = local.environment == "prod" ? [] : [data.aws_ssm_parameter.aws_sso_admin_role.value]
```
The "non_prod" isn't needed. Also the internal buckets the admin user should be removed from the permissions in production

Reply:
Yeah, it looks much simpler. Thanks